sway/commands/bar/colors.c

```diff
@@ -36,7 +36,7 @@ static struct cmd_results *parse_single_color(char **color,
 	}
 
 	char *rgba = hex_to_rgba_hex(argv[0]);
-	if (!*rgba) {
+	if (!rgba) {
```

Comment:
In the same train of thought, there's an unchecked `malloc` that's `snprintf`'d into as part of `hex_to_rgba_hex`. Let's include that as part of this patch so we fix things entirely :)